requirements_test.txt

```diff
@@ -0,0 +1,2 @@
+pytest
+pytest-xdist
```

Comment:
we usually just leave the test requirements in the env.yml and leave them out of the meta.yml. Wouldn't this require an extra install if you wanted to run tests? As it seems it needed to be added to the build script.

Reply:
Sounds good I'll move them back in and will check the meta.yml for dependencies.